ckanext/dcatapit/templates/macros/dcatapit_form_macros.html

```diff
@@ -70,7 +70,7 @@
   {% call input_block(name+ '-key', label or name) %}
 
         <div style="font-size: 14px">
-        {% trans %}Select organization{% endtrans %}
+        {% trans %}The organization selected above will be used as rights holder{% endtrans %}
```

Comment:
Can you pls add this sting to the po/mo files?

Italian label would be "`L'organizzazione proprietaria del dataset, selezionabile nel campo precedente, sarà indicata come titolare.`"